docs/index.md

```diff
@@ -11,32 +11,48 @@ import TokensCard from './src/tokens-card.svg';
 import heroSVG from './src/warp-hero.svg';
 import FoundationsSVG from './src/foundations-card.svg';
 import CompSVG from './src/comp-card.svg';
-const componentData = [
-  {
-    title: 'CSS classes',
-    href: 'foundations/styling/web/unocss',
-    image: { component: ClassesCard, alt: 'A paper with two curly braces.' },
-    description: 'Explore how to use pre-defined utility-first CSS classes'
-  },
-  {
-    title: 'Icons',
-    href: 'components/icons',
-    image: { component: IconsCard, alt: 'Three talk bubbles that are smiling and have closed eyes' },
-    description: 'Browse our library of UI icons'
-  },
-  {
-    title: 'Colors',
-    href: 'foundations/styling/web/background-color#quick-reference',
-    image: { component: ColorsCard, alt: 'Two abstract shapes in grey and black colors, one bigger than the other' },
-    description: 'Get quick access to predefined color palettes for backgrounds, text, and border'
-  },
-  {
-    title: 'Tokens',
-    href: 'foundations/tokens',
-    image: { component: TokensCard, alt: 'Two abstract shapes in grey and black colors, one bigger than the other' },
-    description: 'Discover design tokens for managing color, typography, and spacing'
-  }
-];
+const cardData = {
+  CardType1: [
+    {
+      title: 'CSS classes',
+      href: 'foundations/styling/web/unocss',
+      image: { component: ClassesCard, alt: 'A paper with two curly braces.' },
+      description: 'Explore how to use pre-defined utility-first CSS classes',
+    },
+    {
+      title: 'Icons',
+      href: 'components/icons',
+      image: { component: IconsCard, alt: 'Three talk bubbles that are smiling and have closed eyes' },
+      description: 'Browse our library of UI icons',
+    },
+    {
+      title: 'Colors',
+      href: 'foundations/styling/web/background-color#quick-reference',
```

Comment:
Not part of this PR, but I see that this url no longer reflects the side menu structure (it's called "CSS Classes" now) - just so it's not confusing, maybe we could apply the updated structure before/upon moving the docs to the final repository?